src/lintsampler/lintsampler.py

```diff
@@ -111,160 +116,28 @@ def __init__(
         Attributes
         ----------
         pdf : function
-        seed : None, int, or ``numpy`` random Generator
         vectorizedpdf : boolean
 
         """
-
-        # set args as attributes
+        # set pdf-related parameters as attributes
         self.pdf = pdf
-        self.seed = seed
         self.vectorizedpdf = vectorizedpdf
```

Comment:
For full user protection, we could check that the `vectorizedpdf` is only set if `pdf` satisfies `callable(pdf)==True`.